src/coreclr/inc/regdisp.h

```diff
@@ -407,7 +403,10 @@ inline void SyncRegDisplayToCurrentContext(REGDISPLAY* pRD)
 #elif defined(TARGET_X86)
     pRD->SP         = (DWORD)GetSP(pRD->pCurrentContext);
     pRD->ControlPC  = (DWORD)GetIP(pRD->pCurrentContext);
-#else // TARGET_X86
+#elif defined(TARGET_WASM)
```

Comment:
A nit - all the 32 bit platforms use the same code here, can you please unify it?